build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
         opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
         buildVersionQualifier = System.getProperty("build.version_qualifier", "")
+        kotlin_version = System.getProperty("kotlin.version", "1.8.21")
```

Comment:
Any reason not to use 1.9.22?

Reply:
I'm using the same as that of common utils which has the `AlertingPluginInterface` logic we use here.